### DISCUSSION TITLE
[Idea]: Remove black borders when presenting images.

### DISCUSSION THREAD

**Discussion by bucmado** [sircharlo/meeting-media-manager#2451] [Ideas] [answered]
### Is your idea related to a problem?

N/A

### Describe your idea

The idea would be to display images in full screen. In the event that an image doesn't cover the entire screen, instead of showing black borders, these would be replaced by the same background image, enlarged and with a more subdued color. This would provide a more polished finish when displaying content, similar to the function used in the VCherry system. While VCherry no longer receives support, this feature can be implemented in our system to enhance content presentation.

### Describe alternatives you've considered

N/A

### Additional context

![Screenshot_1](https://github.com/sircharlo/meeting-media-manager/assets/56500218/f8c78713-c78d-47b9-830d-17d455ad6346)
![Screenshot_2](https://github.com/sircharlo/meeting-media-manager/assets/56500218/8134f721-1ef5-4612-a9d5-c29989be873e)
![Screenshot_3](https://github.com/sircharlo/meeting-media-manager/assets/56500218/0d7f3f28-3c95-436d-9071-de2269410559)
![Screenshot_4](https://github.com/sircharlo/meeting-media-manager/assets/56500218/69c370e6-f02e-4bfd-a1d1-373b8d8196f8)
![Screenshot_6](https://github.com/sircharlo/meeting-media-manager/assets/56500218/1193c1b0-0362-458e-9968-ea0fed875281)

**Comment by sircharlo** [comment]
The images provided for meetings fit the 16:9 aspect ratio almost perfectly in most cases. If a local or visiting speaker's images are in an aspect ratio that is distracting, perhaps the best thing to do would be for the speaker himself to find images that are more suited for viewing on televisions. 

Otherwise, are there any practical downsides to having the media displayed as "fit to screen" with a black background?

___

This is no doubt subjective, but I personally find the background behind these three pictures incredibly distracting.

> ![Screenshot_1](https://private-user-images.githubusercontent.com/56500218/324225322-f8c78713-c78d-47b9-830d-17d455ad6346.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM3NTAxODQsIm5iZiI6MTcxMzc0OTg4NCwicGF0aCI6Ii81NjUwMDIxOC8zMjQyMjUzMjItZjhjNzg3MTMtYzc4ZC00N2I5LTgzMGQtMTdkNDU1YWQ2MzQ2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDIyVDAxMzgwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk4YmNjOGZiYzcxNjQ4MmMzNTYzYjQ0M2Y5NjM1ZjE2ZjE4MDRkM2M1MzQ1NDQ5OGI5Yzg4NjNlMmFiNTRhN2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.AdRz57AB_gCFMAmGAPddW-qvXiVrWkqOOn4XTehLKyU) ![Screenshot_2](https://private-user-images.githubusercontent.com/56500218/324225325-8134f721-1ef5-4612-a9d5-c29989be873e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM3NTAxODQsIm5iZiI6MTcxMzc0OTg4NCwicGF0aCI6Ii81NjUwMDIxOC8zMjQyMjUzMjUtODEzNGY3MjEtMWVmNS00NjEyLWE5ZDUtYzI5OTg5YmU4NzNlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDIyVDAxMzgwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTExNmQ3M2I3NGY4MjIyYTcyY2NjMjlkYTgwMTUzNzBlMmY3MjkwNDFlM2U4Y2MwMzE5OGZhYmY0ZmRiMWYwMTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.uqaP_pjPHyamwyQqYm1jgKZOImn_obh9pf3hUAt8fiQ) ![Screenshot_3](https://private-user-images.githubusercontent.com/56500218/324225327-0d7f3f28-3c95-436d-9071-de2269410559.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM3NTAxODQsIm5iZiI6MTcxMzc0OTg4NCwicGF0aCI6Ii81NjUwMDIxOC8zMjQyMjUzMjctMGQ3ZjNmMjgtM2M5NS00MzZkLTkwNzEtZGUyMjY5NDEwNTU5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDIyVDAxMzgwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJlYzRhMDg5OGY5YTQxMjM2N2VlMjFlNGZiNmY5YWY4YTQwZjEwNDkxNDUzMjAzZmY3YThmZWRlYzZhNWNjMjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.PJHLN8b3lj6SJdt17Uxc-41tEzeMoS9A8oMJ9rOdjEc)

**Comment by mtdvlpr** [comment, marked as answer]
@bucmado, thanks for your idea! Besides agreeing with @sircharlo, the current behavior is in line with how JW Library handles presenting images. We try to align with JW Library as much as possible.
